### PR TITLE
update package.json start command to work on Windows

### PR DESCRIPTION
**Pull request by mobiustripper42**
current version of the start: command in package.json does not work on windows computer. 

updated the start command to use the cross-env package to be able to add environmental variables across linux and windows.

does require addition of .npmrc file to use powershell on windows

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "yup": "^0.32.11"
   },
   "scripts": {
-    "start": "GENERATE_SOURCEMAP=false REACT_APP_GIT_HASH=`git rev-parse HEAD` react-scripts start",
+    "start": "cross-env-shell GENERATE_SOURCEMAP=false REACT_APP_GIT_HASH=$(git rev-parse HEAD) react-scripts start",
```
Comment (adamgall) [inline package.json:34]: @tbwebb22 check this out lol

**Review by adamgall — COMMENTED** [review]
just calling out Taylor to look at a much better solution to this problem than we had previously implemented

File: package.json (hunk 1 above)
Comment (tbwebb22) [reply package.json:34]: > @tbwebb22 check this out lol

Ooh this is dope

**Comment by mobiustripper42** [comment]
this solution has stopped working for me, so I am tracking down the problem. closing until solved

**Comment by mobiustripper42** [comment]
fixed readme

**Review by sethhrbek — APPROVED** [review]

**Pull request merged**